frontend/src/components/module-editor/SideBar.tsx

```diff
@@ -37,15 +37,37 @@ import {
   LessonRequest,
   LessonResponse,
 } from "../../APIClients/types/LessonClientTypes";
-import { CourseResponse } from "../../APIClients/types/CourseClientTypes";
+import {
+  CourseResponse,
+  ModuleResponse,
+  CourseRequest,
+  ModuleRequest,
+} from "../../APIClients/types/CourseClientTypes";
 import { formatLessonRequest } from "../../utils/lessonUtils";
+import EditModuleModal from "../common/EditModuleModal";
+import { GET_COURSE } from "../../APIClients/queries/CourseQueries";
 
 const Sidebar = (): React.ReactElement => {
-  const { moduleIndex }: ModuleEditorParams = useParams();
+  const { moduleIndex, courseID }: ModuleEditorParams = useParams();
```

Comment:
It might be nicer to just do the casting up here so that you don't have to cast to `Number` everywhere:
```suggestion
  const { moduleIndex: moduleIndexString, courseID }: ModuleEditorParams = useParams();
  const moduleIndex = Number(moduleIndexString);
```